generate.c

```diff
@@ -1665,8 +1665,16 @@ static walk_rv_t process_symbol_file(char *path, void *arg)
 
 	/* We want to process only .ko kernel modules and vmlinux itself */
 	if (!safe_strendswith(path, ".ko") &&
-	    !safe_strendswith(path, "/vmlinux"))
-		return ret;
+	    !safe_strendswith(path, "/vmlinux")) {
+		if (conf->kernel_dir) {
+			if (conf->verbose)
+				printf("Skip non-object file %s\n", path);
```

Comment:
The reasoning part is "non-object file".  What else do you have in mind for it?

Reply:
The fact that file extension does not match `ko' nor filename does not equal `vmlinux' is not, in my opinion, equivalent to non-object file and can be misleading. File is skipped because it does not conform to naming convention, which should be the printed-out rationale IMO. What do you think?